Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -1154,14 +1158,14 @@ private async Task RemoveUploadAsync(RemoveUploadArgs args, CancellationToken ca
     /// <exception cref="NotSupportedException">The file stream cannot be read from</exception>
     /// <exception cref="InvalidOperationException">The file stream is currently in a read operation</exception>
     /// <exception cref="AccessDeniedException">For encrypted PUT operation, Access is denied if the key is wrong</exception>
-    private async Task<string> PutObjectSinglePartAsync(PutObjectArgs args,
+    private async Task<PutObjectResponse> PutObjectSinglePartAsync(PutObjectArgs args,
         CancellationToken cancellationToken = default)
     {
         //Skipping validate as we need the case where stream sends 0 bytes
         var requestMessageBuilder = await CreateRequest(args).ConfigureAwait(false);
         using var response = await ExecuteTaskAsync(NoErrorHandlers, requestMessageBuilder, cancellationToken);
-        var putObjectResponse = new PutObjectResponse(response.StatusCode, response.Content, response.Headers);
-        return putObjectResponse.Etag;
+        return new PutObjectResponse(response.StatusCode, response.Content, response.Headers,
+            args.ObjectSize.ToString(), args.ObjectName);
```

Comment:
Why is `ObjectSize` being converted to string? Just keep it as a `long`.

Reply:
Done